### DISCUSSION TITLE
Decorators not working

### DISCUSSION THREAD

**Discussion by jasonchrista** [parcel-bundler/parcel#7162] [Q&A]
I have typescript file with a Lit element. I am using a decorator and getting **Unexpected token '@'.** I have `experimentalDecorators: true `in my tsconfig.json. Any ideas?

**Comment by marioml15** [comment]
Hi, I had the same error as you but I solved it changing VSCode configuration:
File => Preferences => Settings => Search ("experimental decorator") and check it.

I solved that problem but now I have another problem which is "requestUpdate();" is not triggering after changing properties values, so my component doesn't rerender.
